__tests__/start.test.js

```diff
@@ -51,9 +78,8 @@ describe('start api', () => {
     global.config.confirmFirstAddress = null;
   });
 
-  it('should start a MultiSig wallet if multisig is true', async () => {
+  it.skip('should start a MultiSig wallet if multisig is true', async () => {
```

Comment:
The `start` mocked tests will need refactoring soon. They only work if the `wallet-id` parameter is exactly `stub_wallet`, so they end up competing with each other for this centralized resource.

If you run the multisig test individually, it should work ok:
```sh
npm run test -- --testNamePattern="should start a MultiSig wallet if multisig is true" --runTestsByPath __tests__/start.test.js
```

Since the issue #191 is already focused on this problem and multisig is not yet in production, I suggest we keep this test skipped for now.

Reply:
With a considerable amount of changes and added complexity I was able to make all the tests run again, but only in `--runInBand` mode.
I consider this to be out of the scope of this PR, but found it relevant to keep the code here for future consideration.
[test__improves_start_tests_with_status_pooling_for_--runInBand_mode.patch.txt](https://github.com/HathorNetwork/hathor-wallet-headless/files/8636356/test__improves_start_tests_with_status_pooling_for_--runInBand_mode.patch.txt)

Reply:
It's not production code. It's just a draft in a direction that we're not currently interested: making the mocked tests less independent.
I'd rather have more discussions about how to to the mocked tests and come up with a new solution when we get to the implementation phase again.